modules/telco-core-scalability.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * telco_ref_design_specs/ran/telco-core-ref-components.adoc
+
+:_content-type: REFERENCE
+[id="telco-core-scalability_{context}"]
+= Scalability
+
+New in this release::
```

Comment:
Like previous comment, this reads like a section that would be in release notes for a specific version, not a design spec, but I may be missing something. If it's not applicable, does it need to be here?

Reply:
I believe I have to stick close to the format of the original and this is in that document.